src/Package.php

```diff
@@ -142,23 +135,36 @@ class Package
      * $package->build();
      * $package->statusIs(Package::INITIALIZED); // true
```

Comment:
The constant name is not correct. This is also true for the `IDLE` one, but I cannot suggest a change for this one via the GitHub UI.

```suggestion
     * $package->statusIs(Package::STATUS_INITIALIZED); // true
```

Reply:
done